### DISCUSSION TITLE
Add horizontal zoom to history panel

### DISCUSSION THREAD

**Discussion by Misiu** [home-assistant/frontend#11251] [Other feature requests]
## The request

History panel got zoom plugin, that was removed some time ago, in https://github.com/home-assistant/frontend/pull/1104.
Without zoom, it is hard to read values, especially if you have many values, like shown here:
![image](https://user-images.githubusercontent.com/1741838/78454025-ba898200-7695-11ea-8645-720bda2af361.png)

I think the zoom plugin should be added, but only to the x-axis.

## The alternatives

Change chart library.
![image](https://user-images.githubusercontent.com/1741838/78454172-aeea8b00-7696-11ea-8356-ea2405390c44.png)
the screen above is from highcharts (https://www.highcharts.com/stock/demo/compare)

**Comment by zsarnett** [comment]
I do like high charts.

**Comment by snaiperr** [comment]
For the hightcharts the 25 bytes only are needed for one point on the graph, but for the standard HA history graph(history integration)  260 byte are used for the 1 point on the graph, thus the highcharts much more effective.

**Comment by Misiu** [comment]
another option is [uPlot](https://github.com/leeoniya/uPlot).
